docs/runtime_options.md

```diff
@@ -40,7 +40,7 @@ Per-container housekeeping is run once on each container cAdvisor tracks. This t
 
 ## Container Hints
 
-Container hints are a way to pass extra information about a container to cAdvisor. In this way cAdvisor can augment the stats it gathers. For more information on the container hints format see its [definition](container/raw/container_hints.go). Note that container hints are only used by the raw container driver today.
+Container hints are a way to pass extra information about a container to cAdvisor. In this way cAdvisor can augment the stats it gathers. For more information on the container hints format see its [definition](https://github.com/google/cadvisor/blob/master/container/common/container_hints.go). Note that container hints are only used by the raw container driver today.
```

Comment:
I'd actually prefer to keep it relative. You just need to make it relative to the docs directory (i.e. `../container/common/container_hints.go`)